components/net/fetch/methods.rs

```diff
@@ -616,6 +621,18 @@ fn should_block_nosniff(request: &Request, response: &Response) -> bool {
     };
 }
 
+/// https://fetch.spec.whatwg.org/#should-response-to-request-be-blocked-due-to-mime-type?
+fn should_block_mime_type(request: &Request, response: &Response) -> bool {
+    let mime_type = response.headers.get::<ContentType>();
+    let csv: Mime = "text/csv".parse().unwrap();
+    request.type_ == Type::Script && mime_type.is_some() && match *mime_type.unwrap() {
```

Comment:
`SubLevel::Csv` doesn't exist, and I already suggested a way to not parse the CSV mimetype below. :)